### ISSUE TITLE
WSL2 with docker-ce: Can't start project with additional_fqdns; ddev.exe must be installed on Windows

### ISSUE THREAD

**Issue by FLeudts** [drud/ddev#5273]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Expected Behavior

A project in a wsl2 system should start if an additional fqn is added. The best solution for me would be if no tests for the correct ip adress would be performed for additional_fqdns or if the right command would be executed with the "wsl2-no-windows-hosts-mgt" flag set.

### Actual Behavior

Because ddev always searches for installations on the windows side in WSL2 it can't start unless manual steps are performed.
Error message:
"Failed to start mediacenter: ddev.exe is not installed on the Windows side, please install it with 'choco install -y ddev'. It is used to manage the Windows hosts file"

If I run the command "ddev config global --wsl2-no-windows-hosts-mgt=true" the project also fails to start.
Error message:
"Failed to start mediacenter: exit status 1"

Manually trying to run the command "sudo --preserve-env=HOME ddev.exe hostname whaloo.agimus 127.0.0.1" also fails. 
Error Message:
"ddev.exe: command not found"

Manually changing the command to "sudo --preserve-env=HOME ddev hostname whaloo.agimus 127.0.0.1" instead works but throws a warning.
Warning Message:
"Not populating custom commands or hostadditions because running with root privileges"

### Steps To Reproduce

1. WSL2 with docker-ce in the Debian installation and without ddev.exe or docker.exe in the windows side
2. add a additional_fqdns to the .ddev/config.yaml
3. ddev start

**Comment by rfay** [comment]
Please install ddev.exe on the Windows side; the standard WSL2 installation does this. It does it so that your hosts file can be managed on the Windows side. It does not require docker or anything else on the Windows side.

**Comment by rfay** [comment]
The idea here is that most people use the browser on the Windows side, so if you add additional_fqdns, they can't resolve unless they get added to the windows-side hosts file. 

Another option you have is to manually add your hostnames to the c:\Windows\system32\drivers\etc\hosts file, but using ddev.exe is easier. 

If you help me understand what happens with `ddev config global --wsl2-no-windows-hosts-mgt=true` (give the full output) we can chase that. But if you're using the browser on the Windows side it won't bring you any happiness because your browser won't be able to look it up.

If you do the standard installation on https://ddev.readthedocs.io/en/latest/users/install/ddev-installation/#windows this is all taken care of for you; please consider doing that.  It won't do any harm to your existing distro, assuming you have an Ubuntu distro and it's the default distro.

**Comment by rfay** [comment]
Did I mention that you can just solve this with `choco install -y ddev` in an admin PowerShell, or by downloading the installer from the DDEV releases page and running it?
